modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/BaseAggregateTest.java

```diff
@@ -490,12 +491,12 @@ public void sumOnDifferentRowsCount(TestAggregateType testAgg) {
                 RootNode<Object[]> root = new RootNode<>(ctx, aggRowType);
                 root.register(aggChain);
 
-                Set<Integer> grpId = IntStream.range(0, grps).boxed().collect(Collectors.toSet());
+                IntSet grpId = IntStream.range(0, grps).boxed().collect(Collectors.toCollection(IntOpenHashSet::new));
```

Comment:
what reason for the change?

Reply:
Just to use primitive collections everywhere in the codebase.